server/index.js

```diff
@@ -5,6 +5,18 @@ const cors = require('cors');
 let app = express();
 app.use(express.json());
 app.use(express.static(__dirname + '/../public/dist'));
+app.use(express.json()); // support json encoded bodies
+app.use(express.urlencoded({ extended: true })); // support encoded bodies
+app.use(cors()); //added for crud routes
+
+//fake user generator
+const fakeUser = (userName) => {
+  let randomNumber = Math.floor(Math.random() * 100 + 1);
+  return {
+    name: userName,
+    imageUrl: `https://hackreactoramazonfrontendcapstone.s3-us-west-2.amazonaws.com/${randomNumber}.jpeg`
+  }
+}
```

Comment:
Might be worth moving this out of index at some point.
A lot of places use a fairly blank default image for new users - having something like that mike make sense if a picture has be attached. Something like that could be stored locally and could improve time down the line.